data/attributes/attributes.go

```diff
@@ -0,0 +1,46 @@
+package attributes
```

Comment:
singular for package names please

Reply:
is there any need to wrap this `Attribute` type into its own `attribute` package?

IMHO it feels more dev friendly to remain at the `metric` one for:
- atomicity: you won't use attributes without metrics
- simplicity: metrics package gets easier to use
- readability/usage: you don't need to type package name all around

Reply:
Sorry I missed this. The reason to abstract it away into its own package is that now it used in both in the `metric` and `event` package. If you want I can keep it all on the `metric` but then the `event` package will have to import `metric` to use the attributes.